pkg/plugins/optional/helm/v1alpha/scaffolds/init.go

```diff
@@ -124,6 +124,29 @@ func (s *initScaffolder) Scaffold() error {
 	return nil
 }
 
+// getNamePrefix will return the value from kustomize config so that we can append
+// in the RBAC rules manifests. If we be unable to find this value we will use
+// the projectName instead.
+func (s *initScaffolder) getNamePrefix() string {
+	filePath := "config/default/kustomization.yaml"
+	content, err := os.ReadFile(filePath)
+	if err != nil {
+		log.Fatalf("failed to read config/default/kustomization.yaml: %s", err)
+	}
+
+	var defaultConfig struct {
+		NamePrefix string `yaml:"namePrefix"`
+	}
+
+	if err := yaml.Unmarshal(content, &defaultConfig); err != nil {
+		log.Warnf("failed to parse kustomization.yaml to get namePrefix: %s", err)
+		log.Warnf("using the project name as a prefix of RBAC and manifests")
+		return s.config.GetProjectName()
```

Comment:
the projectName name needs a trailing `-` as it got removed in the copyFileWithHelmLogic function:

```suggestion
		return fmt.Sprintf("%s-", s.config.GetProjectName()), -1)
```

Reply:
I will test it out